go/contacts/resolution_notice_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package contacts
+
+import (
+	"github.com/keybase/client/go/libkb"
+	"testing"
+
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEncryptContactResolutionForServer(t *testing.T) {
+	// TODO: how do I get this to be logged in? or do I need to move it to
```

Comment:
Can you use `kbtest`

Reply:
we also need an end to end test which i presume will be a pain to write. but important i think to have a basic one.